spec/graphql/mutations/create_club_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Types::MutationType, type: :request do
+  let(:query) { 'createClub' }
+  let(:variables) do
+    {
+      input: {
+        label: Faker::Lorem.word,
+        description: Faker::Lorem.sentence
+      }
+    }
+  end
+
+  it_behaves_like 'with standard user' do
+    it 'creates a new club' do
+      expect { do_graphql_request }.to change(Club, :count).by(1)
+    end
+  end
+
+  it 'creates trigger an error because we are not conencted' do
+    do_graphql_request
+    expect(errors).to be_present
+    expect(errors.dig(0, 'extensions', 'code')).to eq('unauthorized')
+  end
```

Comment:
```suggestion
  describe 'when unauthenticated' do
    before do
      do_graphql_request
    end

    include_examples 'when unauthenticated'
  end
```

Du coup dans le générateur il y a déjà un truc pour tester le cas où tu es pas connecté